sdk/containerregistry/container-registry/review/container-registry.api.md

```diff
@@ -321,6 +320,12 @@ export interface UpdateTagPropertiesOptions extends OperationOptions {
     canWrite?: boolean;
 }
 
+// @public
+export interface UploadBlobOptions extends OperationOptions {
+    // (undocumented)
+    chunkSize?: number;
```

Comment:
I was using the GitHub pull request VS Code extension and somehow accidentally deleted your comment @annelo-msft >.< 

Here it is, recovered from my emails:
> Per our feature crew discussion, we may end up not exposing chunk size in the operation-level API.

Reply:
I will remove the option.